dcc-lsystem/Cargo.toml

```diff
@@ -9,15 +9,71 @@ edition = "2018"
 readme = "../README.md"
 
 [dependencies]
-image = "0.23"
-imageproc = "0.22"
-regex = "1.3"
+image = { version = "0.23", optional = true }
+imageproc = { version = "0.22", optional = true }
+regex = "1.5"
 lazy_static = "1.4"
 dcc-lsystem-derive = { version = "0.1" }
-color_quant = "1.0"
-gif = "0.11"
-tempfile = "3.1"
+# Todo: is color_quant used for anything?
```

Comment:
Looks like it isn't - if the build passes with it removed it's probably fine.

Reply:
ok, removed color_quant